flocker/control/test/test_httpapi.py

```diff
@@ -35,7 +44,25 @@ def initialize(self):
         self.persistence_service.startService()
         self.addCleanup(self.persistence_service.stopService)
 
-    def assertGoodResult(self, method, path, expected_good_result):
+    def assertResponseCode(self, method, path, request_body, expected_code):
```

Comment:
You rely on the result a lot, and it's supposed to be something specific.

1. Rename method to reflect fact that it's doing an action?
2. Docstring.